lib/recurly.d.ts

```diff
@@ -0,0 +1,785 @@
+// Type definitions for recurly
+// Project: recurly
+// Definitions by: Benjamin Eckel <dx@recurly.com>
+
+export as namespace recurly;
+
+export interface Site {
+  id?: string;
+  object?: string;
+  subdomain?: string;
+  publicApiKey?: string;
+  mode?: string;
+  address?: Address;
+  settings?: Settings;
+  features?: string[];
+  createdAt?: Date;
```

Comment:
just to confirm - your API library converts what i assume are ISO string dates from the API into javascript Date objects? as in, these types aren't lies? :)

Reply:
Correct! Thanks for checking.